packages/thirdweb/src/wallets/coinbase/coinbaseWebSDK.ts

```diff
@@ -290,12 +298,22 @@
             accessList: tx.accessList,
             value: tx.value ? numberToHex(tx.value) : undefined,
             gas: tx.gas ? numberToHex(tx.gas) : undefined,
-            from: this.address,
+            from: getAddress(address),
             to: tx.to as Address,
             data: tx.data,
           },
         ],
       })) as Hex;
+
+      trackTransaction({
+        client: client,
+        walletAddress: getAddress(address),
+        walletType: COINBASE,
+        transactionHash,
+        contractAddress: tx.to ?? undefined,
+        gasPrice: tx.gasPrice,
```

Comment:
wonder if we should track the function selector here, we could potentially resolve it to the function name too

Reply:
actually if we move this tracking to `sendTransaction` we would track for all existing and future wallets, and also have things like the functionName. Though we would loose the `walletType` prop.

I wonder if that's worth it though?

Reply:
Yeah,I was thinking we could use the indexer with the transaction hash to get the function information later

Reply:
Yeah. Ok let's try that for now